src/parser/creative_linear_parser.js

```diff
@@ -273,3 +271,24 @@ function parseYPosition(yPosition) {
 
   return parseInt(yPosition || 0);
 }
+
+/**
+ * Getting required attributes from element
+ * @param  {Object} element - DOM element
+ * @param  {Array} attributes - list of attributes
+ * @return {Array|null} null if a least one element not present
+ */
+function getRequiredAttributes (element, attributes) {
+  const values = {};
+  const error = attributes.some((name) => {
```

Comment:
IMO `some` should not have mutations inside, it should just return a boolean.
You should split the assertion and the object assignment